i18n/translations/fr/common.json

```diff
@@ -41,7 +41,7 @@
     "phone": "Entrer un numéro de téléphone valide.",
     "regex": "Veuillez utiliser le bon format.",
     "file-size-too-large": "Sélectionner un fichier plus petit.",
-    "file-size-too-large-all-files": "Les fichiers dépassent le maximum de 6 Mo. Téléversez des fichiers plus petits.",
+    "file-size-too-large-all-files": "Les fichiers dépassent le maximum de 3.5 Mo. Téléversez des fichiers plus petits.",
```

Comment:
Would be nice to pass in the variable for max vs continuing to update this string

Reply:
Actually let's do that in a separate PR

Reply:
will do later